index.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Logger = require('./lib/logger.js')
+
+let singleton
+
+const setupDefaultLogger = function(key, opts) {
+  if (singleton) return singleton
+  singleton = new Logger(key, opts)
+  return singleton
+}
+
+function createLogger(key, options) {
+  return new Logger(key, options)
+}
+
+module.exports = {
+  createLogger
+, setupDefaultLogger
+}
```

Comment:
just revisiting the discussion around unexported `Logger`, shall we add a note to the migration docs?

Reply:
Yep, I added it to the migration docs, and updated the changes in the commit message.  Thanks!